crates/vsock/src/txbuf.rs

```diff
@@ -14,15 +14,17 @@ pub(crate) struct LocalTxBuf {
     head: Wrapping<u32>,
     /// Index into buffer from which data can be added to the buffer
     tail: Wrapping<u32>,
+    pub(crate) conn_tx_buf_size: u32,
 }
 
 impl LocalTxBuf {
     /// Create a new instance of LocalTxBuf.
-    pub fn new() -> Self {
+    pub fn new(conn_tx_buf_size: u32) -> Self {
```

Comment:
`CONN_TX_BUF_SIZE` was a global variable, please use a different name that make sense for `LocalTxBuf`.